common/src/crypto.ts

```diff
@@ -132,11 +135,16 @@ const sortObjKeys = (x: StrObject) =>
       return acc;
     }, {});
 
-const hashJWK = (x: JsonWebKey) =>
-  new Md5().update(JSON.stringify(sortObjKeys(x as StrObject))).toString();
+const hashJWK = (x: JsonWebKey): Sha256 =>
+  new Sha256().update(JSON.stringify(sortObjKeys(x as StrObject)));
```

Comment:
iiuc there is no added benefit to hashing this again, if you use `update` aftewards you can just send back the `stringify` result here no? or jut make an arraybuffer from the string

Reply:
Not a cryptographer, but if I understand correctly hashing both parts before hashing them together is a cryptographic way to ensure both parts are distinct (meaning the last chunk of the first thing can’t be mistaken for the first chunk of the second thing).